README.md

```diff
@@ -27,12 +27,26 @@ Cozy's apps use a standard set of _npm scripts_ to run common tasks, like watch,
 
 ### Fixtures
 
-You can import fixtures to quickly deal with data:
+You can import fixtures to quickly deal with data.
+
+First, make sure you have the latest version of ACH installed
```

Comment:
Why is it necessary? cozy-ach is already in devDep

Reply:
Currently because we call `ACH` directly and not through `yarn ACH` https://github.com/cozy/coachCO2/blob/master/package.json#L22

Reply:
on peut donc le supprimer des devDep je pense tant qu'à faire

Reply:
Doesn't make more sense to keep it and call ACH through `yarn ACH...` ?

Reply:
I have no preference on the matter, I'll let you decide here.

Reply:
donc on devrait changer l'appel via yarn et mettre la lib à jour si nécessaire, mais pas la peine de l'avoir installé globalement